Apps/System/systemapp.c

```diff
@@ -207,18 +220,20 @@ static void about_update_proc(Layer *layer, GContext *nGContext)
                                GTextOverflowModeTrailingEllipsis, GTextAlignmentCenter, 0);
 
     #ifdef PBL_BW
-        graphics_draw_bitmap_in_rect(nGContext, gbitmap_create_with_resource(RESOURCE_ID_REBBLE_LOGO_BW), GRect((bounds.size.w/2)-17, (bounds.size.h/2)-63, 34, 53));
+        graphics_draw_bitmap_in_rect(nGContext, logo_bw, GRect((bounds.size.w/2)-17, (bounds.size.h/2)-63, 34, 53));
         //TODO: Get black and white rocket bitmap for Classic
         //graphics_draw_bitmap_in_rect(nGContext, gbitmap_create_with_resource(RESOURCE_ID_TO_MOON_BW), GRect((bounds.size.w/2)-8, (bounds.size.h/2)+60, 19, 19));
     #else
-        graphics_draw_bitmap_in_rect(nGContext, gbitmap_create_with_resource(RESOURCE_ID_REBBLE_LOGO_DARK), GRect((bounds.size.w/2)-17, (bounds.size.h/2)-63, 34, 53));
-        graphics_draw_bitmap_in_rect(nGContext, gbitmap_create_with_resource(RESOURCE_ID_TO_MOON), GRect((bounds.size.w/2)-8, (bounds.size.h/2)+60, 19, 19));
+        graphics_draw_bitmap_in_rect(nGContext, logo_color, GRect((bounds.size.w/2)-17, (bounds.size.h/2)-63, 34, 53));
+        graphics_draw_bitmap_in_rect(nGContext, rocket_color, GRect((bounds.size.w/2)-8, (bounds.size.h/2)+60, 19, 19));
     #endif    
 }
 
 static void about_window_unload(Window *window)
 {
     scroll_layer_destroy(s_about_scroll);
+    gbitmap_destroy(logo_color);
```

Comment:
Note that this only works on color, and breaks on B&W.  Recommend just having 'logo' and 'rocket', and conditionally initializing them on B&W and color (and assigning them to NULL); then checking for NULL when you destroy.  i.e., 

```c
static GBitmap *logo;
...
logo = create(...);
...
if (logo)
   graphics_draw_bitmap_in_rect...
...
if (logo)
  gbitmap_destroy(logo);
```

Reply:
Great idea, not sure why I didn't think of it. Implemented something similar I think will work, let me know if I need to revise.